.github/linters/.pylintrc

```diff
@@ -135,7 +137,20 @@ disable=print-statement,
         xreadlines-attribute,
         deprecated-sys-function,
         exception-escape,
-        comprehension-escape
+        comprehension-escape,
+        invalid-name,
+        broad-except,
+        superfluous-parens,
+        missing-module-docstring,
```

Comment:
Is there already a pylint RC file used for the Pecan project? Disabling some of these warnings appears to reduce code quality, reuse, and understanding. For example, ignoring missing documentation and inconsistent-return-statements

Reply:
Actually, there were a lot of warnings for inconsistent-return-statements, missing-module-docstring and somehow pylint action treats them as an error, not warning and so it fails the test for linting.
So should I remove them from pylint RC and open a separate issue for that?

Reply:
Here's an issue for that #70

Reply:
great - thanks!